hierarchicalforecast/utils.py

```diff
@@ -68,7 +68,106 @@ def cov2corr(cov, return_std=False):
     else:
         return corr
 
+# %% ../nbs/utils.ipynb 7
+# convert levels to output quantile names
+def level_to_outputs(level):
+    """ Converts list of levels into output names matching StatsForecast and NeuralForecast methods.
+
+    **Parameters:**<br>
+    `level`: int list [0,100]. Probability levels for prediction intervals.<br>
+
+    **Returns:**<br>
+    `output_names`: str list. String list with output column names.
+    """
+    qs = sum([[50-l/2, 50+l/2] for l in level], [])
+    output_names = sum([[f'-lo-{l}', f'-hi-{l}'] for l in level], [])
+
+    sort_idx = np.argsort(qs)
+    quantiles = np.array(qs)[sort_idx]
+
+    # Add default median
+    quantiles = np.concatenate([np.array([50]), quantiles])
+    quantiles = torch.Tensor(quantiles) / 100
```

Comment:
Currently, the library does not need `torch` as a dependency. Is there a particular reason to return and use `quantiles` as a torch tensor? Using the function could throw errors since `torch` is not being imported in this module. Changing these lines to 

```
quantiles = np.concatenate([np.array([50]), quantiles]) / 100
```

should work.

Reply:
fixed, thanks!